.gitignore

```diff
@@ -9,5 +9,9 @@ web/profiles/contrib
 # Ignore Drupal's file directory
 web/sites/default/files
 
+# Ignore environment specific settings
+web/sites/default/services.yml
```

Comment:
Do we really want to ignore services.yml? I've yet to see anything sensitive like API keys in it.